keps/sig-node/606-compute-device-assignment/README.md

```diff
@@ -79,6 +81,16 @@ As such the external monitoring agents need to be able to determine the set of d
 
 ![device monitoring architecture](https://user-images.githubusercontent.com/3262098/43926483-44331496-9bdf-11e8-82a0-14b47583b103.png)
 
+#### Device aware CNI plugin
+
+As soon as this interface has been introduced it was used by CNI plugins like [kuryr-kubernetes](https://review.opendev.org/#/c/651580/) in couple with [intel-sriov-device-plugin](https://github.com/intel/sriov-network-device-plugin) to correctly define which devices were assigned to the pod.
+
+#### Topology aware scheduling
+
+This interface can be used to collect allocated resources with information about the NUMA topology of the worker node.
```

Comment:
Since distribution of available devices are continuously changing (newly launched pod consumes it), the goal is to be informed which device from which numa node was used. As was written above, but w/o details intel-sriov-device-plugin provides pci address, which could be used by CNI plugin to detect NUMA id, but it was not a general solution. The solution in general way is numa id (topology) in container devices.
So to be more specific, this sentence could be:
This interface can be used to track down resources with information about the NUMA topology of the worker node in general way.

Reply:
It seems like there's a specific use case here! Would you mind detailing it as an example?